src/metakb/cli.py

```diff
@@ -363,9 +662,9 @@ def _retrieve_s3_cdms() -> str:
         msg = "Unable to locate files matching expected resource pattern in VICC s3 bucket"
         raise FileNotFoundError(msg)
 
-    echo_info(f"Retrieved CDM files dated {newest_version}")
+    _echo_info(f"Retrieved CDM files dated {newest_version}")
     return newest_version
 
 
 if __name__ == "__main__":
-    update_metakb_db(_anyio_backend="asyncio")
```

Comment:
not totally sure why the asyncio thing was needed? maybe an old click version? I've been able to run the async `transform` command without problems.